packages/css/src/index.js

```diff
@@ -129,12 +129,14 @@ export const responsive = styles => theme => {
 
   for (const key in styles) {
     const value = styles[key]
+    if (value === null || value === undefined) continue
```

Comment:
also the same as `if (value == null)` but you probably know that, I prefer this in libs